message_structure.md

```diff
@@ -15,18 +15,23 @@ The following example shows common elements of a DIDComm Message. Further detail
     "created_time": 1516269022,
     "expires_time": 1516385931,
     "body": {
-    	"messagespecificattribute": "and it's value"
+    	"messagespecificattribute": "and its value"
 	}
 }
 ```
 
 ### Message Headers
 
-The attributes of a DIDComm Message at the level of its outer packaging (effectively, the "headers" of the message) are as follows:
+The attributes of a DIDComm plaintext message at the level of its outermost object (effectively, its "headers") are as follows:
 
 - **id** - REQUIRED. Message ID. The `id` attribute value MUST be unique to the sender.
 - **type** - REQUIRED. Message Type. The `type` attribute value MUST be a valid [Message Type URI](protocols.md#message-type-uri) , that when resolved gives human readable information about the message. The attributes value also informs the content of the message, for example the presence of other attributes and how they should be processed.
-- **to** - OPTIONAL. Recipient(s) identifier. The `to` attribute MUST be an array of strings where each element is a valid [DID](https://w3c.github.io/did-core/#generic-did-syntax) which identifies the recipients of the message.
+- **to** - OPTIONAL. Identifier(s) for recipients. MUST be an array of strings where each element is a valid [DID](https://w3c.github.io/did-core/#generic-did-syntax) that identifies a member of the message's intended audience.
+
```

Comment:
indicate what is implied if the to attribute is missing.

Reply:
Done.